src/liballoc/boxed.rs

```diff
@@ -816,6 +816,26 @@ impl From<Box<str>> for Box<[u8]> {
     }
 }
 
+#[stable(feature = "box_from_array", since = "1.44.0")]
```

Comment:
Thanks to my slow review, it's 1.45.0 now °_°

Reply:
Yes, I noticed that, I was just waiting for the review.